pkg/apis/config/testing/defaults.go

```diff
@@ -0,0 +1,24 @@
+package testing
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	logtesting "knative.dev/pkg/logging/testing"
+)
+
+// SetDefaults sets the default ConfigMap values in an existing context (for use in testing)
+func SetDefaults(ctx context.Context, t *testing.T, data map[string]string) context.Context {
```

Comment:
thanks for adding the helper function! would you mind moving the refactoring into a separate PR? other than that this LGTM

Reply:
Sure, created a separate PR and updates the usage of the new helper function PTAL!: https://github.com/tektoncd/pipeline/pull/6339

/hold unitil https://github.com/tektoncd/pipeline/pull/6339 is merged

Reply:
https://github.com/tektoncd/pipeline/pull/6339 is merged

/hold cancel